io/EptReader.cpp

```diff
@@ -710,8 +712,10 @@ PointViewSet EptReader::run(PointViewPtr view)
 
             if (m_info->dataType() == EptInfo::DataType::Laszip)
                 startId = readLaszip(*view, key, nodeId);
-            else
+            else if (m_info->dataType() == EptInfo::DataType::Binary)
                 startId = readBinary(*view, key, nodeId);
+            else
+                startId = readZstandard(*view, key, nodeId);
```

Comment:
Seems like an #ifdef for ZSTD here would be cleaner.

Reply:
77890cf36cfe98559fc6975cc4fc48d37d673470